docs/authentication/saml/jit-provisioning.mdx

```diff
@@ -0,0 +1,17 @@
+---
+description: Learn about Clerk's support for Just-In-Time account provisioning with all supported SAML providers.
+---
+
+# JIT account provisioning with SAML
+
+Just-in-Time (JIT) account provisioning, or automatic account provisioning, is a process by which accounts for employees are created on-demand when they sign-up to an app with SAML using a company email address.
+
+Using JIT account provisioning means your IT department won't have to manually create user accounts for each of the services or apps your employees use to get work done. Consider using this feature if you don't need to control who accesses your Clerk application on a case-by-case basis.
```

Comment:
This is supported by default and customers don't have a way to opt-out for the time being
```suggestion
Using Just-in-Time (JIT) Provisioning means your IT department won't have to manually create user accounts for each of the services or apps your employees use to get work done.
```